--- MySQL-Sandbox-3.0.42/lib/MySQL/Sandbox/Scripts.pm.orig	2013-09-30 03:59:30.000000000 -0500
+++ MySQL-Sandbox-3.0.42/lib/MySQL/Sandbox/Scripts.pm	2013-10-21 09:28:31.506402691 -0500
@@ -1359,7 +1359,7 @@
 
     'grants.mysql'  => <<'GRANTS_MYSQL',
 
-use mysql;
+USE mysql;
 set password=password('_DBPASSWORD_');
 grant all on *.* to _DBUSER_@'_REMOTE_ACCESS_' identified by '_DBPASSWORD_';
 grant all on *.* to _DBUSER_@'localhost' identified by '_DBPASSWORD_';
